esx_service/vmci/vmci_client.c

```diff
@@ -60,6 +60,8 @@ typedef struct be_request {
 
 typedef struct be_answer {
    char *buf;   // calloced, so needs to be free()
+   char *errBuf;   // calloced, so needs to be free()
```

Comment:
Can you elaborate on the comment ? Why calloced buffer does not need to be free ?